chitter.rb

```diff
@@ -0,0 +1,57 @@
+require 'sinatra/base'
+require './lib/message.rb'
+require './lib/user.rb'
+require 'pg'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+    
+  get '/' do
+    redirect('/messages')
+  end
+
+  get '/messages' do
+    @list = Message.all
+    @user = session[:user]
+    #@authors = @messages.each { |message| message.author }
```

Comment:
A couple of very minor points: a comment left in and a couple of 'p's left in - otherwise this is great; all core user stories completed. 
One missing function is password; maybe do some research into the encryption gem mentioned in last week's challenge (bcrypt)